include/mbedtls/platform.h

```diff
@@ -43,6 +43,8 @@
 #include "platform_time.h"
 #endif
 
+#define MBEDTLS_ERR_PLATFORM_HW_FAILED       -0x0080 /**< Hardware platform function failed. */
```

Comment:
All of our HW failure error codes have the suffix of `HW_ACCEL_FAILED`.
For example[`MBEDTLS_ARIA_HW_ACCEL_FAILED`](https://github.com/ARMmbed/mbedtls/blob/development/include/mbedtls/aria.h#L52)

This is not a HW accelerator, but more of a platform HW failure, so I removed the "ACCEL" part.

Since the error code was already merged to Mbed OS, unless this is a blocker, I prefer to keep the same error code.

Unless your comment was about the error comment, then I'll change

Reply:
I'll change, but this error is for the platform function, not a generic hardware failure.

How about " Hardware platform operation failure." ?